tests/foreman/api/test_host.py

```diff
@@ -441,144 +531,208 @@ def test_positive_create_with_usergroup(self):
+        assert host.content_facet_attributes['lifecycle_environment_id'] == module_lce_search.id
+
+        host.content_facet_attributes = {
+            'content_view_id': module_cv_with_puppet_module.id,
+            'lifecycle_environment_id': module_lce_search.id,
+        }
+        host = host.update(['content_facet_attributes'])
+        assert host.content_facet_attributes['content_view_id'] == module_cv_with_puppet_module.id
+        assert host.content_facet_attributes['lifecycle_environment_id'] == module_lce_search.id
 
     @tier1
-    def test_positive_create_with_host_parameters(self):
+    def test_positive_end_to_end_with_host_parameters(self, module_org, module_location):
         """Create a host with a host parameters specified
+        then remove and update with the newly specified parameters
 
         :id: e3af6718-4016-4756-bbb0-e3c24ac1e340
 
-        :expectedresults: A host is created with expected host parameters
+        :expectedresults: A host is created with expected host parameters,
+        paramaters are removed and new parameters are updated
 
         :CaseImportance: Critical
         """
         parameters = [{'name': gen_string('alpha'), 'value': gen_string('alpha')}]
         host = entities.Host(
-            organization=self.org, location=self.loc, host_parameters_attributes=parameters
+            organization=module_org,
+            location=module_location,
+            host_parameters_attributes=parameters,
         ).create()
-        self.assertEqual(host.host_parameters_attributes[0]['name'], parameters[0]['name'])
-        self.assertEqual(host.host_parameters_attributes[0]['value'], parameters[0]['value'])
-        self.assertIn('id', host.host_parameters_attributes[0])
+        assert host.host_parameters_attributes[0]['name'] == parameters[0]['name']
+        assert host.host_parameters_attributes[0]['value'] == parameters[0]['value']
+        assert 'id' in host.host_parameters_attributes[0]
+
+        parameters = []
+        host.host_parameters_attributes = parameters
+        host = host.update(['host_parameters_attributes'])
+        assert host.host_parameters_attributes == []
+
+        parameters = [{'name': gen_string('alpha'), 'value': gen_string('alpha')}]
+        host.host_parameters_attributes = parameters
+        host = host.update(['host_parameters_attributes'])
+        assert host.host_parameters_attributes[0]['name'] == parameters[0]['name']
+        assert host.host_parameters_attributes[0]['value'] == parameters[0]['value']
+        assert 'id' in host.host_parameters_attributes[0]
 
     @tier2
-    def test_positive_create_with_image(self):
-        """Create a host with an image specified
+    def test_positive_end_to_end_with_image(
+        self, module_org, module_location, module_cr_libvirt, module_libvirt_image
+    ):
+        """Create a host with an image specified then remove it
+        and update the host with the same image afterwards
```

Comment:
I suspect (the lack of) indent on the continuation line causes travis issue in all three tests

Reply:
exactly :)